eZ/Publish/Core/Persistence/Legacy/Content/Type/Handler.php

```diff
@@ -401,8 +401,17 @@ public function copy($userId, $contentTypeId, $status)
         $createStruct->modifierId = $userId;
         $createStruct->created = $createStruct->modified = time();
         $createStruct->creatorId = $userId;
-        $createStruct->remoteId = substr(sha1(uniqid(get_class($createStruct), true)), 0, 7);
-        $createStruct->identifier = 'cpy_' . $createStruct->identifier . '_' . $createStruct->remoteId;
+        $originalRemoteId = $createStruct->remoteId;
+        // truncate remoteId to 32 chars to keep BC length of that field
+        $createStruct->remoteId = substr(sha1(uniqid(get_class($createStruct), true)), 0, 32);
```

Comment:
Memory is not the issue, it's the BC break on do schema. Is truncated sha1 still unique?